docs/buildah-commit.md

```diff
@@ -39,13 +51,21 @@ Default leaves the container and its content in place.
 
 ## EXAMPLE
 
-buildah commit containerID
+This example saves an image based on the container.
+ `buildah commit containerID`
+
+This example saves an image named newImageName based on the container.
+ `buildah commit --rm containerID newImageName`
+
 
-buildah commit --rm containerID newImageName
+This example saves an image based on the container disabling compression and specifying a signature policy (not recommended).
+ `buildah commit --disable-compression --signature-policy '/etc/containers/policy.json' containerID`
```

Comment:
Why are we specifying --signature-policy, isn't this the default?  I actually thought this was a hidden field.  Specifying in each example, makes it seem like something useful for a user to do, but we expect them to almost never do it.

Reply:
@nalind thoughts on this.  I primarily just added the comment over each and the one new example for the auth.

Reply:
It is the default.  The option is documented higher up with a note that its use is not recommended, so that the system-wide default will be used.  Removing it from the examples should be fine.

Reply:
ok, I'll remove it from all but one of the examples here then.

Reply:
Actually I lied, I'll remove it entirely from the examples.